app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsFragment.kt

```diff
@@ -558,7 +558,8 @@ class MergedAppcoinsFragment : DaggerFragment(), MergedAppcoinsView {
     var fiatText = formatter.formatCurrency(fiatAmount, WalletCurrency.FIAT)
         .plus(" $currency")
     if (isSubscription && frequency != null) {
```

Comment:
If it is a subscription, it must have a frequency. We can also avoid de null validation and enforcing if we set a default value for when the frequency is missing instead of using `null`